schema.sql

```diff
@@ -42,6 +26,19 @@ create table "public"."quizzes" (
     "friendly_id" character varying
 );
 
+alter table "public"."quizzes" add column "views" bigint default '0'::bigint;
+CREATE OR REPLACE FUNCTION public.increment(x integer, row_id integer)
+ RETURNS integer
+ LANGUAGE sql
+ SECURITY DEFINER
+AS $function$
+  update quizzes 
+  set views = views + x
+  where id = row_id;
+
+  select views from quizzes where id = row_id ;
+$function$
+;
```

Comment:
In its current state schema.sql actually fails execution on a fresh db. I had to move these alterations down in the script to a point where the quizzes table has already been created.